cockpit-podman.spec.in

```diff
@@ -11,7 +11,7 @@ BuildRequires:  libappstream-glib
 
 Requires:       cockpit-bridge >= 138
 Requires:       cockpit-shell >= 138
-Requires:       podman >= 1.3.0
+Requires:       podman >= 2.0.0
```

Comment:
OOI, is that really strictly 2.0, or are the 1.9.x versions previews/betas for what will become the final 2.0? I. e. 1.9.3 is currently in Fedora and IIRC that doesn't suffice, but will 1.9.4?